tests/test_autodiscover.py

```diff
@@ -7,16 +7,10 @@
 
 class TestAutoDiscover:
     def test_autodiscover(self):
-        health_check_plugins = list(filter(
-            lambda x: 'health_check.' in x and 'celery' not in x,
-            settings.INSTALLED_APPS
-        ))
+        health_check_plugins = list(
+            filter(lambda x: x.startswith("health_check."), settings.INSTALLED_APPS)
+        )
```

Comment:
list comprehension seems to be more readable in this case:

 ```suggestion
        health_check_plugins = [
            app_name
            for app_name in settings.INSTALLED_APPS
            if app_name.startswith("health_check.")
        ]
```

Reply:
The file was formatted with black